### ISSUE TITLE
Add "Create clj-kondo hook" actions

### ISSUE THREAD

**Issue by mrkam2** [clojure-lsp/clojure-lsp#1737]
**Is your feature request related to a problem? Please describe.**
I'd like to automate the following, currently manual, steps I need to take to create a new clj-kondo hook:
1. Update `deps.edn` file to include `"resources"` folder into `:paths` if it isn't there yet.
2. Create `resources/clj-kondo.export/<my-org>/<my-lib>/config.edn` file if it doesn't exist yet.
3. Update this file content to assoc-in `[:hooks :analyze-call <fully-qualified-macro-name>] hooks.<fully-qualified-macro-name>` for analyze-call hooks and the same with `:macroexpand` instead of `:analyze-call` for the hook of the macro expand type.
4. Create `resources/clj-kondo.export/<my-org>/<my-lib>/hooks/<namespace path>/<namespace name>.clj` file.
5. Add a new definition to this file - for `:analyze-call` it should be 
  ```clojure
  (defn <macro-name> [{:keys [node]}]
    (let [[<macro-args>] (rest node)]
       ;; TODO Add your code here.
       {:node node}))
  ```
  For `:macroexpand` it should be
  ```clojure
  (defmacro <macro-name> [<macro-args>]
     ;; TODO Add you code here.
     )
  ```

**Describe the solution you'd like**
I'd like the following workflow. Open a macro in editor, invoke either "create analyze-call hook" or "create macroexpand hook" and have all of these steps performed and my cursor jump to `Add your code here` line. 

It would also be great to have a command to jump from a macro to its hook and back (and maybe to its line in clj-kondo config file).

Reference:
https://github.com/clj-kondo/clj-kondo/blob/master/doc/hooks.md#hooks

**Comment by ericdallo** [comment]
You may be interested on this one @borkdude

**Comment by borkdude** [comment]
When exporting configs, don't use `hooks/whatever` for your clj-kondo namespaces, but use `my_company/whatever`, to avoid conflicts with others.

**Comment by mrkam2** [comment]
> When exporting configs, don't use `hooks/whatever` for your clj-kondo namespaces, but use `my_company/whatever`, to avoid conflicts with others.

Makes sense. Sounds like we might want to repeat the same `<my-org>/<my-lib>` if those aren't already part of an original namespace, which makes the path a bit repetitive, especially considering the full path. Something like:
`my-company/cool-lib/resources/clj-kondo.exports/my-company/cool-lib/my-company/cool-lib/feature/helper.clj`.

I'm wondering if we were able to reduce this to something like this:
`my-company/cool-lib/resources/clj-kondo.exports/hooks/feature/helper.clj`.

Do you think this may be changed in the future?

**Comment by borkdude** [comment]
This won't be changed. The part after `clj-kondo.exports` is only what clj-kondo sees.

**Comment by borkdude** [comment]
Just check what ends up in `.clj-kondo` and it should be clear how clashes can happen.
